main/resource-access.bs

```diff
@@ -130,6 +130,88 @@ Note:
 Servers that wish to disable URI re-use may want to use the `410` status
 code.
 
+### Resource type heuristics ### {#resource-type-heuristics}
+
+Servers MUST apply the following heuristics to determine a request's resource
+type
+[[Source](https://github.com/solid/specification/issues/128#issuecomment-573033297)].
```

Comment:
Consider explaining under what circumstances the server needs to fall back to this heuristic approach. (i.e. when the client does not specify an interaction model)?